perceval/client.py

```diff
@@ -79,18 +76,29 @@ class HttpClient:
     GET = "GET"
     POST = "POST"
 
-    def __init__(self, base_url, max_retries=MAX_RETRIES, status_forcelist=DEFAULT_STATUS_FORCE_LIST,
-                 default_sleep_time=DEFAULT_SLEEP_TIME, headers=DEFAULT_HEADERS):
+    def __init__(self, base_url, max_retries=MAX_RETRIES, default_sleep_time=DEFAULT_SLEEP_TIME,
+                 extra_headers={}, extra_status_forcelist=[], extra_retry_after_status=[]):
```

Comment:
This is an error. We can't initialize default values that way because they are mutable.

See http://docs.python-guide.org/en/latest/writing/gotchas/